src/Zoo.cpp

```diff
@@ -0,0 +1,239 @@
+#include "..\include\Zoo.h"
+#include "Lion.h"
+#include "Monkey.h"
+#include "Shark.h"
+
+
+void Zoo::run()
+{
+	int index;
+	initZoo();
+	std::string type;
+	std::string name;
+	bool validCommand;
+
+	std::string userInput = "";
+	while (!(userInput == "exit")) {
+		printMap();
+		printList();
+		printMenue();
+		std::cin >> userInput;
+		std::cout << "you entered:  " << userInput << std::endl;
+		validCommand = true;
+		if (userInput == "stop")
+		{
+			std::cout << "enter the animal index:" << "\n";
+			std::cin >> index;
```

Comment:
you can put the input / output command in the functions, and make the run function shorter